matrix/all-in-one.yaml

```diff
@@ -0,0 +1,534 @@
+component: Splunk Distribution of OpenTelemetry .NET
+version: 1.0.2
+dependencies:
+  - link: https://github.com/open-telemetry/opentelemetry-dotnet
+    version: 1.6.0
+    stability: stable
+  - link: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation
+    version: 1.0.2
+    stability: stable
+settings:
+  - name: SPLUNK_TRACE_RESPONSE_HEADER_ENABLED
+    description: Activated by default. Adds server trace information to HTTP response headers. For more information, see :ref:`server-trace-information-dotnet-otel`. The default value is `true`.
+    default: true
```

Comment:
it should be string

```suggestion
    default: "true"
```

same problem applies in other places

Reply:
3f7228db8662bccf3940d80344dfce19a2a7211d